app/client/components/billing/invoicesTable.tsx

```diff
@@ -35,7 +34,7 @@ interface InvoicesTableProps {
 export const InvoicesTable = (props: InvoicesTableProps) => {
   const [
     trackingPaginationInteractionCount,
-    setTrackingPaginationInteractionCount
+    setTrackingPaginationInteractionCount,
```

Comment:
I believe our linter removes trailing commas. Could you check if the linter ran correctly on this file before merging?